frontend/src/concepts/pipelines/content/compareRuns/CompareRunsRunList.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { ExpandableSection } from '@patternfly/react-core';
+import { TableVariant } from '@patternfly/react-table';
+import { PipelinesFilter } from '~/concepts/pipelines/types';
+import { Table, useCheckboxTable } from '~/components/table';
+import CompareRunTableRow from '~/concepts/pipelines/content/compareRuns/CompareRunTableRow';
+import { compareRunColumns } from '~/concepts/pipelines/content/compareRuns/columns';
+import DashboardEmptyTableView from '~/concepts/dashboard/DashboardEmptyTableView';
+import usePipelineFilter, {
+  FilterOptions,
+  getDataValue,
+} from '~/concepts/pipelines/content/tables/usePipelineFilter';
+import CompareRunTableToolbar from '~/concepts/pipelines/content/compareRuns/CompareRunTableToolbar';
+import { useCompareRuns } from '~/concepts/pipelines/content/compareRuns/CompareRunsContext';
+
+const CompareRunsRunList: React.FC = () => {
+  const { runs, loaded } = useCompareRuns();
+  const [isExpanded, setExpanded] = React.useState(true);
+  const [, setFilter] = React.useState<PipelinesFilter | undefined>();
+  const filterToolbarProps = usePipelineFilter(setFilter);
+  const {
+    tableProps: checkboxTableProps,
+    toggleSelection,
+    isSelected,
+  } = useCheckboxTable(runs.map(({ run_id: runId }) => runId));
+
```

Comment:
Sorry, i thought i added this comment.

but you will need to create a custom useCheckboxTable. Some other pipelines tables need to in order to not have selections stored in this hook. Becuase we have them in our context

```tsx
const useCompareRunsCheckboxTable = (): UseCheckboxTable => {
  const { selectedRuns, setSelectedRuns, runs } = useCompareRuns();
  return useCheckboxTableBase<PipelineRunKFv2>(
    runs,
    selectedRuns,
    setSelectedRuns,
    React.useCallback((d) => d.run_id, []),
  );
};
```
Something like this.